src/test/java/Homework23.java

```diff
@@ -0,0 +1,57 @@
+import PageObjectModel.HomePage;
+import PageObjectModel.LoginPage;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class Homework23 extends BaseTest {
+
+    String newNameForPlaylist = "test";
+    @Test
+    public void renameTestPlaylist() throws InterruptedException {
+
+        String updatedPlaylistMsg = "Updated playlist \"test.\"";
+
+        LoginPage loginPage = new LoginPage(driver);
+        HomePage homePage = new HomePage(driver);
+
+        loginPage.provideEmailToLogin("lolitamantsiuk@gmail.com")
+                 .providePasswordToLogin("te$t$tudent")
+                 .clickSubmitBtn();
+        Thread.sleep(2000);
+        homePage.doubleClickOnSelectedPlaylist();
+        Thread.sleep(2000);
+        homePage.enterNewNameForPlaylist();
+        Thread.sleep(2000);
+        getRenamedPlaylistSuccessMsg();
+        Thread.sleep(2000);
+        Assert.assertEquals(getRenamedPlaylistSuccessMsg(),updatedPlaylistMsg);
+    }
```

Comment:
Please, delete `throws InterruptedException`

**Thread. sleep is bad! It blocks the current thread and renders it unusable for further work.
Using Thread. sleep() frequently in an automation framework is not a good practice.**

Please use the Explicit Wait command, the WebDriver is directed to wait until a certain condition occurs before proceeding with executing the code.